### PR TITLE
Adding gym-saturation

### PR DESCRIPTION
**Pull request by inpefess**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by inpefess** [comment]
I confirm I'm willing to be listed as a maintainer of the `gym-saturation` package

**Comment by inpefess** [comment]
PR is ready for review

**Comment by inpefess** [comment]
@conda-forge-admin, please ping team

File: recipes/gym-saturation/meta.yaml
Comment (BastianZim) [inline recipes/gym-saturation/meta.yaml:32]: It seems like these should be under `run` as they are required by the package: https://github.com/inpefess/gym-saturation/blob/f1e113b1bd798dea6de7f7ca10a3b6021d330d9f/pyproject.toml#L27-L28

File: recipes/gym-saturation/meta.yaml
Comment (BastianZim) [inline recipes/gym-saturation/meta.yaml:23]: Upstream lists `importlib_resources`: https://github.com/inpefess/gym-saturation/blob/f1e113b1bd798dea6de7f7ca10a3b6021d330d9f/pyproject.toml#L29

Is that not required?

File: recipes/gym-saturation/meta.yaml
Comment (BastianZim) [inline recipes/gym-saturation/meta.yaml:21]:
```suggestion
    - poetry-core >=1.0.0           
```

**Review by BastianZim — CHANGES_REQUESTED** [review]

**Comment by inpefess** [comment]
@BastianZim Thank you for your suggestions, I've commited fixes.

**Comment by ocefpaf** [comment]
@BastianZim you were the first reviewer here so I'll defer to you? Ready for merge?

**Comment by BastianZim** [comment]
Sorry, you were faster than me I was just double checking everything. :)

Yes, should be good to go!

**Pull request merged**
